test/unittests/wasm_engine_test.cpp

```diff
@@ -5,6 +5,63 @@
 using namespace fizzy;
 using namespace fizzy::test;
 
+TEST(wasm_engine, parse_error)
+{
+    const auto wasm = "0102"_bytes;
+
+    for (auto engine_create_fn : {create_fizzy_engine, create_wabt_engine, create_wasm3_engine})
+    {
+        auto engine = engine_create_fn();
+        ASSERT_FALSE(engine->parse(wasm));
+    }
+}
+
+TEST(wasm_engine, DISABLED_instantiate_error)
```

Comment:
Why is this disabled?

Reply:
Need to fix #138 to make this pass on wasm3. And wabt had some other issue I can't remember.

Reply:
I can enable this test for fizzy only, if that helps.

Reply:
I'm not sure we can have generically parse and instantiate separated. You can leave it for fizzy only.